packages/dds/tree/src/test/simple-tree/unhydratedNode.spec.ts

```diff
@@ -177,6 +177,24 @@ describe("Unhydrated nodes", () => {
 		assert.equal(Tree.status(object), TreeStatus.New);
 	});
 
+	it("preserve events after hydration - minimal", () => {
+		const log: unknown[] = [];
+		const leafObject = new TestLeaf({ value: "value" });
```

Comment:
Just a note for clarity/posterity: it initially confused me to see events being subscribed to on (what I thought was) a leaf. Then I realized `TestLeaf extends schemaFactory.object(...)` so it's really an object node. Just noting my confusion; maybe we want to use a different name instead of `TestLeaf` for scenarios like this.